<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools::FspPrinterDialogConds</class>
 <widget class="QWidget" name="Tools::FspPrinterDialogConds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QCheckBox" name="nouveau">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>si vous êtes le nouveau médecin traitant cochez cette case</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Accident causé par un tiers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="accident">
       <item>
        <property name="text">
         <string>NC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>non</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>oui</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Utils::ModernDateEditor" name="accidentDate"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="mater">
       <property name="text">
        <string>Maternité</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>date début de grossesse ou accouchement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Utils::ModernDateEditor" name="materDate"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="maladie">
       <property name="text">
        <string>Maladie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="exo">
       <item>
        <property name="text">
         <string>Pas d'exoneration du ticket modérateur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exoneration du ticket modérateur</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="ald">
       <property name="text">
        <string>ALD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="prev">
       <property name="text">
        <string>Acte de prévention</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="l115">
       <property name="text">
        <string>L115</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autre">
       <property name="text">
        <string>Autre</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="at">
       <property name="text">
        <string>AT/MP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numéro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="atNb"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Utils::ModernDateEditor" name="atDate"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="envoyeur"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>si le patient est envoyé par le médecin traitant, complétez la ligne ci-dessous nom et prénom du médecin traitant :</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>s'il ne l'est pas, sélectionnez un item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="parcours">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accès direct spécifique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Urgence</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hors résidence habituelle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Médecin traitant remplace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accès hors coordination</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ModernDateEditor</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/moderndateeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
